fetch-core-dump.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+# get sensor pod name
+sensor_pod_name=$(kubectl -n stackrox get pods -l app=sensor -o custom-columns=NAME:.metadata.name | tail -n 1)
+
+# Create core dump
+kubectl exec ${sensor_pod_name} -- /bin/bash -c "cd /var/cache/stackrox && /bin/bash -c 'gcore 1'"
+
+# Copy core-dump
+kubectl cp ${sensor_pod_name}:/var/cache/stackrox/core.1 ./core.1
```

Comment:
Let's maybe add some names to it and add the quotes
```suggestion
kubectl cp "${sensor_pod_name}:/var/cache/stackrox/core.1" "core-dump.1.${sensor_pod_name}"
```

Reply:
done